server/models/admin.sql

```diff
@@ -194,4 +194,36 @@ CREATE PROCEDURE zMergeAccounts(
   DELETE FROM account WHERE id = from_account_id;
 END $$
 
+/*
+CALL zComputeAllInventoryStockQuantities(startDate, depotUuid)
+
+DESCRIPTION
+Recomputes the quantity in stock for all inventory items for a given depot.
+
+*/
+DROP PROCEDURE IF EXISTS zComputeAllInventoryStockQuantities $$
+CREATE PROCEDURE zComputeAllInventoryStockQuantities (
+  IN _start_date DATE,
+  IN _depot_uuid BINARY(16)
```

Comment:
Aren't we going to need an end date too?   For instance, we may need to compute these things over a month but may be several days into the next month.